qp_shotgun/humann2/__init__.py

```diff
@@ -62,7 +62,8 @@
     'input-format': ['choice:["", "fastq", "fastq.gz", "fasta", "fasta.gz", '
                      '"sam", "bam", "blastm8", "genetable", "biom"]', ''],
     'pathways': ['choice:["metacyc", "unipathway"]', 'metacyc'],
-    'memory-use': ['choice:["minimum", "maximum"]', 'minimum']}
+    'memory-use': ['choice:["minimum", "maximum"]', 'minimum'],
+    'remove-column-description-output': ['boolean', 'True']}
```

Comment:
If we pass false to this parameter it will break the pipeline right? In that case I don't think we should expose this parameter.